articles/active-directory-b2c/add-password-reset-policy.md

```diff
@@ -159,6 +163,8 @@ A claims transformation technical profile initiates the **isForgotPassword** cla
 
 The **SelfAsserted-LocalAccountSignin-Email** technical profile **setting.forgotPasswordLinkOverride** defines the password reset claims exchange that executes in your user journey.
 
+The **LocalAccountWritePasswordUsingObjectId** technical profile **UseTechnicalProfileForSessionManagement** AAD session manager is required for sessions to persist correctly for SSO.
```

Comment:
Can you review this statement @bolt-io

Reply:
Also, AAD might confuse readers as they expect something to do with Azure AD B2C, so you can omit it.

Reply:
Hey @kengaderdus - agreed, I have used the explicit `SM-AAD` session manager reference to avoid confusion. This now reads: 

"The **LocalAccountWritePasswordUsingObjectId** technical profile **UseTechnicalProfileForSessionManagement** `SM-AAD` session manager is required for the user to preform subsequent logins successfully under [SSO](./custom-policy-reference-sso.md) conditions."

Please let me know of any further refinement